Workbooks/WorkbooksMetadata.json

```diff
@@ -1142,15 +1142,15 @@
     "provider": "IntSights Cyber Intelligence"
   },
   {
-    "workbookKey": "DarktraceSummaryWorkbook",
+    "workbookKey": "AIADarktraceSummaryWorkbook",
     "logoFileName": "Darktrace.svg",
     "description": "A workbook containing relevant KQL queries to help you visualise the data in model breaches from the Darktrace Connector",
     "dataTypesDependencies": [ "CommonSecurityLog" ],
-    "dataConnectorsDependencies": [],
-    "previewImagesFileNames": [ "DarktraceSummaryWhite.png", "DarktraceSummaryBlack.png" ],
+    "dataConnectorsDependencies": [ "AIA-Darktrace" ],
+    "previewImagesFileNames": [ "AIA-DarktraceSummaryWhite.png", "AIA-DarktraceSummaryBlack.png" ],
     "version": "1.0",
```

Comment:
If there is any update in workbook then we  have to increase the version number. Could you please increase the version number.

Thanks,
Aditi Maheshwari